Dockerfile

```
@@ -1,43 +1,46 @@
-FROM ubuntu:xenial-20181113
+FROM quay.io/bedrock/ubuntu:22.04
 
-MAINTAINER "René Moser" <mail@renemoser.net>
+ENV DEBIAN_FRONTEND noninteractive
 
-ARG src_url=https://github.com/apache/cloudstack/archive/4.13.1.0.tar.gz
+ARG src_url=https://github.com/apache/cloudstack/archive/refs/tags/4.17.2.0.tar.gz
 
 RUN echo 'mysql-server mysql-server/root_password password root' | debconf-set-selections; \
     echo 'mysql-server mysql-server/root_password_again password root' | debconf-set-selections;
 
-RUN apt-get -y update && apt-get dist-upgrade -y && apt-get install -y \
+RUN apt-get -qq update && apt-get -qq dist-upgrade && apt-get install -qq -y --no-install-recommends \
```

I just noticed that this step is taking quite a long time. With the `-qq` it's impossible to see what's taking so long.

Can we drop the `-qq` and also omit the `dist-upgrade`?

```suggestion
RUN apt-get update && apt-get install -y --no-install-recommends \
```

---

Perhaps AZP was just slow to show the output. Looking at the run for the previous commit it seems this step was less than 3 minutes, and it did provide progress details. Other build steps were responsible for taking the bulk of the time.